nerfstudio/model_components/renderers.py

```diff
@@ -335,47 +335,46 @@ def forward(
         ray_samples: RaySamples,
         ray_indices: Optional[Int[Tensor, "num_samples"]] = None,
         num_rays: Optional[int] = None,
-    ) -> Float[Tensor, "*batch 1"]:
+        quantity: Optional[Float[Tensor, "*batch num_samples 1"]] = None,
```

Comment:
What are your thoughts on this @FrederikWarburg? I think maybe we should have a separate renderer?

Reply:
So the main differences are:

1. rename steps -> quantity
2. add the argument quantity.

Don't think these changes justify a separate renderer?